### ISSUE TITLE
Botania 1.8 compat

### ISSUE THREAD

**Issue by williewillus** [Azanor/thaumcraft-5#335]
**Tldr**: Since my postInit is happening before yours, my potion integration is binding to null and crashing since you instantiate potions in postInit. Is there a way for me to get the potions earlier, or postInit after you?

---

Hey, the Botania 1.8 porter here, couldn't catch you on IRC lately.

Recently, Thaumcraft added integration with Botania. However, it specified a after:Botania dependency which led to a mod loading loop. In response, we removed our after:Thaumcraft dependency.

However, after some investigation, I actually need to load after Thaumcraft. Botania adds several brews and items that integrate with Thaumcraft potion effects, which don't actually seem to be instantiated until your postInit :/

Since B is before T, Botania postInits first, binds to null potions since Thaumcraft hasn't instantiated them yet, and the integration crashes later on.

Is there any pressing reason TC needs to load after Botania, or any reason why potions are done in postInit instead of init? In short, I need a way to access your potions during my postInit, which is happening before yours currently.

Thanks!!

**Comment by Azanor** [comment]
Well I don't actually specify any sort of loading order on my side, so I guess it might just be a result of the forge API dependency thing. I could try specifying before:botania on my side, but I'm not sure if it will be overridden by forge. Alternatively I could remove the api package file on my side. That might work, though it could become messy if the api changes on your side. I only have IPetalApothecary in my side so I guess that is not likely?
